builder/xenserver/iso/builder.go

```diff
@@ -93,6 +107,7 @@ func (self *Builder) Prepare(raws ...interface{}) (params []string, retErr error
 		pargs["apic"] = "true"
 		pargs["timeoffset"] = "0"
 		pargs["acpi"] = "1"
+		pargs["cores-per-socket"] = "1"
```

Comment:
Do we not want this to be a configurable option? Why do we need to explicitly add it if we are setting a default of 1?